nixos/modules/services/monitoring/netdata.nix

```diff
@@ -213,6 +213,7 @@ in {
         ++ lib.optional cfg.python.enable (pkgs.python3.withPackages cfg.python.extraPackages)
         ++ lib.optional config.virtualisation.libvirtd.enable (config.virtualisation.libvirtd.package);
       environment = {
+        NETDATA_PIPENAME = "/tmp/netdata-ipc";
```

Comment:
Are you sure the pipe is there?

Reply:
https://github.com/netdata/netdata/blob/2c305f3bac39a5e404f988921f1587fa67dd12b5/daemon/pipename.c#L8

Reply:
How did you test this PR?

Reply:
> How did you test this PR?
```shell
while [ ! -e "$NETDATA_PIPENAME" ] || [ "$(${cfg.package}/bin/netdatacli ping)" != "pong" ]; do
  if [ -e "$NETDATA_PIPENAME" ]; then echo "$NETDATA_PIPENAME"; fi
  sleep 0.5
done
```